README.md

```diff
@@ -14,7 +14,12 @@ your machine.
 
 ## Installation
 
-Install this package using [Composer](https://getcomposer.org/):
+
+### PHAR
```

Comment:
The [PHP documentation](https://secure.php.net/manual/en/intro.phar.php) as well as [Composer](https://getcomposer.org/download/) write "phar" as a lowercase word. I think we should do the same (or "Phar" in this case because it is at the start of a sentence).